### PR TITLE
Divide the loss by update_freq when using gradient accumulation.

### PR DESCRIPTION
**Pull request by stephenroller**
**Patch description**
Presently in ParlAI, gradient accumulate just continuously sums gradients into the appropriate `grad` tensors. However, to be roughly more congruent with simply increasing the batch size, fairseq and others actually *average* over the gradient accumulation. This ensures the semantics of a learning rate stay *roughly* fixed over different levels of gradient accumulation.

**Testing steps**
Been using on a private branch for a few weeks now. Doesn't make a huge difference, but I prefer to be congruent with others.

### CODE REVIEW

**Comment by stephenroller** [comment]
Seems reasonable to me. Changing the default.
